runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -267,15 +272,33 @@ else if (player != client.getLocalPlayer() && (player.isClanMember() || player.i
 	@Subscribe
 	public void onPlayerLootReceived(final PlayerLootReceived playerLootReceived)
 	{
-		if (config.screenshotKills())
+		if (config.screenshotKills() && !isAtLMS() && !client.getWorldType().contains(WorldType.TOURNAMENT))
```

Comment:
As mentioned previously, this would prevent screenshots of player kills on DMM tournament worlds, which players would likely not appreciate. Additionally, a test case for this specific scenario (DMM tournament world) should be added to ensure it is working as expected.